File: src/main/java/com/aagu/blog/controller/DashboardController.java
package com.aagu.blog.controller;

import com.aagu.blog.aspect.Auth;
import com.aagu.blog.service.ArticleService;
import com.aagu.blog.service.CommentService;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.HashMap;
import java.util.Map;

@RestController
@RequestMapping("/admin")
class DashboardController {
    private final CommentService commentService;

    private final ArticleService articleService;

    public DashboardController(CommentService commentService, ArticleService articleService) {
        this.commentService = commentService;
        this.articleService = articleService;
    }

    @GetMapping("/dashboard/statistic")
    @Auth
    public Object statistic() {
        Map<String, String> resp = new HashMap<>();
        resp.put("visitor","120");
        resp.put("comment", String.valueOf(commentService.getCommentCount()));
        resp.put("article", String.valueOf(articleService.getArticleCount()));
        resp.put("resource", "8");
        return resp;
    }
}

File: src/main/java/com/aagu/blog/Controllers/DashboardController.kt
package com.aagu.blog.Controllers

import com.aagu.blog.Services.ArticleService
import com.aagu.blog.Services.CommentService
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/admin")
class DashboardController {
    @Autowired
    lateinit var commentService: CommentService

    @Autowired
    lateinit var articleService: ArticleService

    @GetMapping("/dashboard/statistic")
    fun statistic(): Any {
        val visitorCount = 120
        val commentCount = commentService.getCommentCount()
        val articleCount = articleService.getArticleCount()
        val resourceCount = 8;
        return mapOf(
                Pair("visitor", visitorCount),
                Pair("comment", commentCount),
                Pair("article", articleCount),
                Pair("resource", resourceCount)
        )
    }
}